src/test/regress/output/multi_alter_table_statements.source

```diff
@@ -844,6 +844,156 @@ SELECT relname FROM pg_class WHERE relname LIKE 'lineitem_alter%' ORDER BY relna
 (15 rows)
 
 \c - - - :master_port
+-- verify that we can set and reset storage parameters
+ALTER TABLE lineitem_alter SET(fillfactor=40);
```

Comment:
We should also verify these commands are correctly propagated when creating new shards (or moving one), for example by running them on a local table and then running create_distributed_table.

For the commands to be applied to a new shard, master_get_table_ddl_events needs to return the appropriate command.

Reply:
This is now tested too.